### PR TITLE
Migrated Prometheus metrics for Kafka

### LINKED ISSUE #569: Migrate Kafka metrics through Prometheus integration
# What should we add to our content collections?

(more information is useful here! Were you trying to do something particular?)
Migrate Kafka metrics through Prometheus integration https://help.aiven.io/en/articles/3298562-kafka-metrics-through-prometheus-integration

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Migrated Article "Kafka metrics through Prometheus integration"

https://help.aiven.io/en/articles/3298562-kafka-metrics-through-prometheus-integration

Fix #569

### CODE REVIEW

**Comment by laysauchoa** [comment]
I noticed the `help` label, what help do you need? or how can others help this PR?

**Comment by ftisiot** [comment]
I believe the HELP is to get someone with hands on prometheus experience to validate

**Comment by lornajane** [comment]
Exactly what @ftisiot said, I'm not familiar with the integration so it would be nice if someone who had used it could comment

**Comment by minh-hoang-aiven** [comment]
The metrics `kafka_consumer_group_*` are made available from us but have not been updated to the doc

**Comment by ftisiot** [comment]
Hi @minh-hoang-aiven are you still planning to work on this?

**Comment by minh-hoang-aiven** [comment]
> Hi @minh-hoang-aiven are you still planning to work on this?

I made a commit last week. Just waiting for a review now

File: docs/products/kafka/reference/kafka-metrics-prometheus.rst [hunk 1]
```diff
@@ -1,7 +1,18 @@
 List of metrics available via Prometheus integration
 ==================================================================
 
-Below you can find a summary of every metric available via Prometheus for an Aiven for Apache Kafka service.
+The list of available metrics highly depends on the service usage, hence, there are a few metrics would be available to
```
Comment (ftisiot) [inline docs/products/kafka/reference/kafka-metrics-prometheus.rst:4]: Why some metrics should be available or not? Does this depends on the components used?

File: docs/products/kafka/reference/kafka-metrics-prometheus.rst [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+certain services while they cannot be found in other services. The following list only contains the most common metrics
+available via Prometheus for an Aiven for Apache Kafka service.
+
+You can retrieve the complete list of available metrics for your specific service by calling to the Prometheus endpoint
+
+.. code-block:: python
```
Comment (ftisiot) [inline docs/products/kafka/reference/kafka-metrics-prometheus.rst:10]: this is not python, maybe bash?

**Review by ftisiot — COMMENTED** [review]
Good work, I would change the minor things mentioned and then ask the review from one of our solutions architects

**Comment by ftisiot** [comment]
Hi @minh-hoang-aiven, are you done with the editing? I still see some comments open.

Thanks!

**Comment by minh-hoang-aiven** [comment]
> Hi @minh-hoang-aiven, are you done with the editing? I still see some comments open.
> 
> Thanks!

Let me close this PR and make another PR with the complete doc to clean it up a bit.

**Pull request closed without merging**